sdk/communication/communication-phone-numbers/test/utils/index.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PhoneNumberCapabilitiesRequest, PhoneNumberCapabilityValue } from "../../src";
+
+export const buildCapabilityUpdate = (
+  capabilities: PhoneNumberCapabilitiesRequest
+): PhoneNumberCapabilitiesRequest => {
+  const values: PhoneNumberCapabilityValue[] = ["none", "inbound", "outbound", "inbound+outbound"];
+
+  return {
+    sms: shuffle(values).find((val) => val != capabilities.sms),
+    calling: shuffle(values).find((val) => val != capabilities.calling)
+  };
+};
+
+// Fisher-Yates Shuffle algo implementation
```

Comment:
Haha, overkill but okay. Used to be one of the easier interview questions I asked.